packages-web/badge-button/src/BadgeButton.webmodeler.tsx

```diff
@@ -0,0 +1,41 @@
+import { createElement, ReactElement } from "react";
+import { BadgeButton } from "./components/BadgeButton";
+import { BadgeButtonPreviewProps } from "../typings/BadgeButtonProps";
+
+declare function require(name: string): string;
```

Comment:
Include node type 😅

Reply:
No let not do that. let keep it this way. Maybe we have to look into using a different tsconfig, also to prevent `logger.debug("oopsy")` killing the preview